utils/chains.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanche-network-runner/utils/constants"
+	coreth_params "github.com/ava-labs/coreth/params"
+)
+
+func LoadGenesisMap() (map[string]interface{}, error) {
```

Comment:
I supposed a function that should return only the new content for the field cChainGenesis, not the full genesis. because it is also beign modified on local after that, I imagine will be best to all genesis updated to be present at local, and this hiddes the first genesis change